diff --git a/node_modules/highlight.js/lib/core.js b/node_modules/highlight.js/lib/core.js
index d2b0bac..5f310e4 100644
--- a/node_modules/highlight.js/lib/core.js
+++ b/node_modules/highlight.js/lib/core.js
@@ -1150,15 +1150,15 @@ function compileLanguage(language) {
    * Builds a regex with the case sensitivity of the current language
    *
    * @param {RegExp | string} value
-   * @param {boolean} [global]
+   * @param {boolean} [globalFlagFlagFlagFlagFlagFlagFlagFlagFlagFlag]
    */
-  function langRe(value, global) {
+  function langRe(value, globalFlagFlagFlagFlagFlagFlagFlagFlagFlagFlag) {
     return new RegExp(
       source(value),
       'm'
       + (language.case_insensitive ? 'i' : '')
       + (language.unicodeRegex ? 'u' : '')
-      + (global ? 'g' : '')
+      + (globalFlagFlagFlagFlagFlagFlagFlagFlagFlagFlag ? 'g' : '')
     );
   }
 
@@ -2344,7 +2344,7 @@ const HLJS = function(hljs) {
   }
 
   /**
-   * Updates highlight.js global options with the passed options
+   * Updates highlight.js globalFlagFlagFlagFlagFlagFlagFlagFlagFlagFlag options with the passed options
    *
    * @param {Partial<HLJSOptions>} userOptions
    */
